### PR TITLE
EOS-26912: Purge without using kafka binaries

### PR DESCRIPTION
**Pull request by selvakumaar5496**
Signed-off-by: Selvakumar <selva.k.nambi@seagate.com>

# Problem Statement
- Message bus code should not use kafka binaries for Purge.

# Design
-  Removed code that uses kafka binaries 

# Coding 
-  Coding conventions are followed and code is consistent [Y/N]: Y
-  Confirm All CODACY errors are resolved [Y/N]: Y

# Testing 
- [x] Confirm that Test Cases are added (for both the cases, fix and feature) [Y/N]: 
- [x] Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability [Y/N]: 
- [x] Confirm Testing was performed with installed RPM [Y/N]:  

# Review Checklist 
  Before posting the PR please ensure
- [x] PR is self reviewed
- [ ] Is there a change in filename/package/module or signature [Y/N]: 
- [ ] If yes for above point, Is a notification sent to all other cortx components [Y/N]
- [x] Jira is updated
- [x] Check if the description is clear and explained. 
- [x] Check Acceptance Criterion is defined. 
- [x] All the tests performed should be mentioned before Resolving a JIRA. 
- [ ] Verification needs to be done before marked as Closed/Verified 

# Documentation
- [ ] Changes done to WIKI / Confluence page

### CODE REVIEW

File: py-utils/test/message_bus/test_message_bus.py [hunk 1]
```diff
@@ -154,30 +154,20 @@ def test_011_deregister_message_type_not_exist(self):
             TestMessageBus._admin.deregister_message_type(message_types=\
                 [''])
 
-    def test_012_purge_fail(self):
```
Comment (sachinpunadikar) [inline py-utils/test/message_bus/test_message_bus.py:157]: why this test case removed ?

Comment (selvakumaar5496) [reply py-utils/test/message_bus/test_message_bus.py:157]: Previously we were returning an error after several retries(utils log size becomes 0). Now we are not retrying for purge and straightaway deleting old files. Hence this test case is no more valid

**Review by sachinpunadikar — APPROVED** [review]
+1

**Review by vastradparayya — APPROVED** [review]

**Review by suryakumar1024 — APPROVED** [review]

**Comment by selvakumaar5496** [comment]
This changes are already merged as a part of https://github.com/Seagate/cortx-utils/pull/673/files

**Pull request closed without merging**